src/icons/IconPlayIndoor.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+
+type Props = { className?: string };
+
+export default function IconPlayIndoor({ className = '' }: Props) {
+  return (
+    <svg
+      width="24px"
+      height="24px"
+      viewBox="0 0 24 24"
+      version="1.1"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <title>play_inside_icon</title>
+      <g
+        id="Currents"
+        stroke="none"
+        stroke-width="1"
```

Comment:
This is still giving error: "Warning: Invalid DOM property `stroke-width`. Did you mean `strokeWidth`?"